tcs-persistence/src/main/java/com/transformuk/hee/tis/tcs/service/model/Post.java

```diff
@@ -87,6 +87,8 @@
   private Set<Placement> placementHistory = new HashSet<>();
   @OneToMany(mappedBy = "post", fetch = FetchType.LAZY)
   private Set<PostTrust> associatedTrusts;
+  @OneToMany(mappedBy = "post", fetch = FetchType.EAGER, cascade = CascadeType.REMOVE)
```

Comment:
I'm a little wondering if the FetchType can be set to Lazy because I saw the `placementEsrEvents` in Placement class is.

Reply:
Good point, let me do a bit of checking 🔬

Reply:
Ah, this doesn't seem to affect things (though I'd agree LAZY fetch is better, all other things being equal). But I did notice something else in the JSON serialization - see Post.java comment below!